### PR TITLE
BC-4528 - Release 32.1 - Version updates

### PR DESCRIPTION
**Pull request by bergatco**
# Description

PR to update the `dof_app_deploy` to the latest release versions.

## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-4528

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/group_vars/all/config.yml [hunk 1]
```diff
@@ -484,7 +484,7 @@ configuration_all:
   FEATURE_COLUMN_BOARD_ENABLED:
     value: "false"
     server: true
-    client: true
+    client: false
```
Comment (dyedwiper) [inline ansible/group_vars/all/config.yml:487]: Why this change? Because the value is just not needed in the old client?

Comment (bergatco) [reply ansible/group_vars/all/config.yml:487]: That's correct. The old client has nothing to do with the board and therefore has no need of this variable.

**Review by dyedwiper — APPROVED** [review]
